membership_policy.md

```diff
@@ -239,6 +243,24 @@ After an extended period away from the project with no activity
 those members would need to re-familiarize themselves with the current state
 before being able to contribute effectively.
 
+### Inactive Reviewers/Approvers/Chairs/Leads
+
+Reviewers, approvers, chairs and leads are pillars of the community and are fundamental to maintaining the momentum of the project.
+Considering that they have elevated permissions and responsibilities in the project, their active engagement with the project is greatly appreciated.
+
+If a reviewer and/or approver steps down from their position in a repository or the project entirely, we trust that they will notify the community and create PRs removing them as reviewers/approvers in the applicable repositories.
+This will help others identify and grow into the role so as to not leave a gap for any length of time. 
+It will also ensure they are not erroneously tagged to review/approve PRs, which is both an inconvenience to them but also an obstacle for the project.
+
+When a SIG or WG Chair or Subproject Lead needs to step down from their position, it is expected that they work with the SIG/Subproject/WG to identify and mentor a willing candidate to grow into the role.
+They should then open a PR to replace them in [sigs.yaml], and move themselves to emeritus status (if applicable).
+
+A reviewer/approver/chair/lead may also be demoted or removed from their role if they are inactive in their area for 6 months or more.
```

Comment:
I think we also should think about the case of a "vanishing" person. In that case there's no candidate, so the impacted group or the community should select one.

This could happen maybe by approaching the candidate - either by maintainers of the affected group or the project maintainers in the worst case - and then suggesting them to the group or community in a PR against the `sigs.yaml`. A quorum of people with authority should then be enough to accept the suggestion.

Reply:
+1.
In general I like the idea of having an "escape hatch" to such defined policies. After all, it makes sense that due to circumstances sometimes someone will vanish with no notice and won't have the time to fulfill this process or even communicate the situation. In such a scenario, I would say that if the majority of SIG chairs vote to remove them because they know they won't be active for the near future, there is no need to wait for 6 months just to get the "proof" of inactiveness.

No policy can replace common sense 100%, but IMO is mostly there to serve as a general guideline.